src/core/util/debug.js

```diff
@@ -1,9 +1,11 @@
+/* @flow */
+
 import config from '../config'
 import { noop } from 'shared/util'
 
 export let warn = noop
 export let tip = noop
-export let formatComponentName
+export let formatComponentName: Function = (null: any) // work around flow check
```

Comment:
This cast is necessary because we cannot persuade flow that formatComponentName is initialized before other code.